### DISCUSSION TITLE
更新后使用WiFi就提示失败开4g就不会

### DISCUSSION THREAD

**Discussion by Li114381** [2dust/v2rayNG#3523] [General]
使用WiFi后测试速度为-1
底下测试就弹出失败： net/http: TLS handshake timeout
使用数据流量就完全不会 
![Screenshot_2024-08-24-13-36-21-805_com v2ray ang](https://github.com/user-attachments/assets/99a981f5-664c-4a33-a9c0-cb8730a4eeee)
![Screenshot_2024-08-24-13-36-27-460_com v2ray ang](https://github.com/user-attachments/assets/3dcc15a5-e8c8-4081-845a-3dfef9dbb9c3)
![Screenshot_2024-08-24-13-36-32-282_com v2ray ang](https://github.com/user-attachments/assets/67b15e8e-23fc-40da-b23c-8b6d436a3f3e)
![Screenshot_2024-08-24-13-36-37-277_com v2ray ang](https://github.com/user-attachments/assets/52937d8f-f4a1-46dd-b078-48e57b854c2c)
![Screenshot_2024-08-24-13-36-39-502_com v2ray ang](https://github.com/user-attachments/assets/270d5280-8821-4627-94d7-45c602f79bbe)

**Comment by 2640395850** [comment]
我也有这个问题，突然就连不上了，昨天更新了服务器上的证书，修改了端口都不行，看到这个问题去试了一下，才发现使用流量能正常走代理

**Reply by devxnan** [reply]
我也有这个问题，突然就连不上了，用流量可以，使用 wifi 就会报 net/http: TLS handshake timeout。
